data-sources/tidbcloud/config.yml

```diff
@@ -0,0 +1,25 @@
+# A unique identifier for the data source
+id: tidbcloud
+
+# Display name for the data source
+displayName: TiDB Cloud Integration
+
+description: |
+  New Relic's TiDB Cloud quickstart provides performance insights including database time, queries per second, and resource utilization. Install the quickstart to better understand utilization of resources and monitor performance issues.
+
+install:
+  primary:
+    link:
+      url: https://docs.pingcap.com/tidbcloud/monitor-newrelic-integration
```

Comment:
I am not sure how to write this URL, so I included a document explaining it that TiDBCloud will work with to release.

Reply:
Sorry I think I'm not understanding. Are we waiting for this link to work? Because right now it points to a 404

Reply:
@lilyjazz this URL goes to a Page Not Found error.  Please update the URL to point to a page with the appropriate content

Reply:
Sorry for the late reply. The link is indeed not yet live, but I have prepared a PR for the documentation. Once the PR is merged on our New Relic repo, it will be synced and go live. Is that okay for you?
The documentation PR link: https://github.com/pingcap/docs/blob/7ced9df3c25c77320aa0c6ae11b4a26189d55765/tidb-cloud/monitor-new-relic-integration.md

Reply:
> @brammerl @seemantk @zstix Hi, guys. What else do I need to do for this pull request? Can we merge it?

Thank you for your patience and diligence here.  Apparently, I needed to sign the CLA (?), which I've done as well.  I'm approving this for merging by the hero on April 4